### DISCUSSION TITLE
is the DeepXDE scale-dependent?

### DISCUSSION THREAD

**Discussion by mlmamud** [lululxvi/deepxde#793] [Q&A]
Hi, is the DeepXDE scale-dependent? I tried to use DeepXDE for a simple 1-D forward problem like this one dy^2/dx^2 = 0. I found that for a small domain [0, 1] DeepXDE gives good results without using data inside the domain from the reference solution. But, it did not predict a good result with larger domain [0, 5], [0, 10], or [0, 100]. Even for the 1-D domain [0, 100], deepXDE could not predict good result with 500 data from the reference solution. Following are simulations of some cases to make you understand my problem. 

I also would like to know, can we use the installed DeepXDE package for domain discretization? So far, my understanding is that the user can only give geometry input and DeepXDE discretize the domain based on boundary values in input data from the reference solution. 

**Case-1:** 
Domain = [0, 1]
data = dde.data.PDE(geom, pde, bc, 0, 2, solution=func, num_test= 100)
![image](https://user-images.githubusercontent.com/67887543/178611712-21214530-ddd3-432e-993a-0128b7a85fc0.png)

**Case-2:** 
Domain = [0, 5]
data = dde.data.PDE(geom, pde, bc, 0, 2, solution=func, num_test= 100)
![image](https://user-images.githubusercontent.com/67887543/178611903-6b4bc08d-01df-4966-90b7-c1821ee6891b.png)

**Case-3:** 
Domain = [0, 5]
data = dde.data.PDE(geom, pde, bc, 5, 2, solution=func, num_test= 100)
![image](https://user-images.githubusercontent.com/67887543/178611964-42425b64-0c9d-4565-af52-7356592997db.png)

**Case-4:** 
Domain = [0, 100]
data = dde.data.PDE(geom, pde, bc, 0, 2, solution=func, num_test= 100)
![image](https://user-images.githubusercontent.com/67887543/178612111-e1ef0e31-655a-4c8b-9fcf-74402f87e418.png)

**Case-5:** 
Domain = [0, 100]
data = dde.data.PDE(geom, pde, bc, 500, 2, solution=func, num_test= 100)
![image](https://user-images.githubusercontent.com/67887543/178612163-72f3a52d-2092-47fc-9f8e-baec0a0b3043.png)

**Comment by lululxvi** [comment]
- See FAQ "Q: I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses."
- Yes.

**Reply by mlmamud** [reply]
Thank you so much for you reply!

**Reply by MINE0126** [reply]
Hello, Do you know that any paper discusses this problem? Thanks.

**Reply by mlmamud** [reply]
> Hello, Do you know that any paper discusses this problem? Thanks.

Hello, I do not have any paper that discussed the issue. We figured out that tanh activation function plays the role. That is why most of the PINN work's spatial domain is 0-1. Thanks.
